### PR TITLE
✨ Source Zendesk Sell : Migrate Python CDK to Low-code CDK

### LINKED ISSUE #30887: :sparkles: Source Zendesk Sell : Migrate Python CDK to Low-code CDK
https://docs.airbyte.com/integrations/sources/zendesk-sell

**Rules for the Connectors Hackathon**
You can only work on one connector at a time
Make sure to resolve all suggestions from the Airbyte team
Must include all streams of the connector you’re migrating. Otherwise, it will not be accepted.
If you don’t commit anything within five days, the issue may be assigned to someone else.

### PR DESCRIPTION
**Pull request by aazam-gh**
## What
Fix #30887 
Migrate Source Zendesk Sell from Python CDK to Low-code CDK

## How
Using Low Code/No Code Connector Development Kit

## Recommended reading order
1. `manisfest.yaml`
2. `schema`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Thanks for the contribution 🚀 . I added it to the team backlog and we're planning to start next week the review process.

File: airbyte-integrations/connectors/source-zendesk-sell/build.gradle [hunk 1]
```diff
@@ -0,0 +1,8 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+}
+
+airbytePython {
+    moduleDirectory 'source_zendesk_sell'
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-sell/build.gradle:7]: Remove this file. It isn't used anymore.

File: airbyte-integrations/connectors/source-zendesk-sell/integration_tests/configured_catalog.json [hunk 2]
```diff
@@ -217,4 +217,4 @@
       "destination_sync_mode": "overwrite"
     }
   ]
-}
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-sell/integration_tests/configured_catalog.json:220]: Please fix the EOF for all files.

File: airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_token'] }}"
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/manifest.yaml:21]: This is not corresponding to existing connector where you have a specific `next_page_token` function. Please implement correct pagination.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Some comments. Missing doc changelog update

File: airbyte-integrations/connectors/source-zendesk-sell/build.gradle (hunk 1 above)
Comment (aazam-gh) [reply airbyte-integrations/connectors/source-zendesk-sell/build.gradle:7]: Done

**Review by marcosmarxm — APPROVED** [review]
Thanks @aazam-gh

**Pull request merged**